src/Datadog.Trace.ClrProfiler.Managed/Integrations/WebRequestIntegration.cs

```diff
@@ -82,7 +155,10 @@ public static object GetResponse(object webRequest, int opCode, int mdToken, lon
                 return callGetResponse(webRequest);
             }
 
-            using (var scope = ScopeFactory.CreateOutboundHttpScope(Tracer.Instance, request.Method, request.RequestUri, IntegrationId, out var tags))
+            // Try to extract existing span context from headers
+            var spanContext = SpanContextPropagator.Instance.Extract(request.Headers.Wrap());
```

Comment:
Why are we looking for propagation headers in an _outbound_ request? Don't we just need to inject headers?

Reply:
See the answer to your other comment

Reply:
I see. Can we add some code comments saying we generated the span id and added the headers earlier but delayed creating the `Scope`?